src/SaveHandler/MongoDB.php

```diff
@@ -57,8 +57,10 @@ public function __construct($mongo, MongoDBOptions $options)
     {
         if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo)) {
             throw new InvalidArgumentException(
-                'Parameter of type %s is invalid; must be MongoClient or Mongo',
-                (is_object($mongo) ? get_class($mongo) : gettype($mongo))
+                sprintf(
+                    'Parameter of type %s is invalid; must be MongoClient or Mongo',
+                    (is_object($mongo) ? get_class($mongo) : gettype($mongo))
+                )
```

Comment:
Personally, I think the way you've suggested indenting it is what *caused* this error to occur.  It's obvious to me that someone's written their code, and copy paste/modified the arguments within, and not noticed the `sprintf` tucked away :)